### PR TITLE
Probe.php:  some work for standardization

### PR DESCRIPTION
**Pull request by rabuzarus**
- rename class methods to camelCase
- apply coding standards
- rename some variables - so they are more comprehensible

### CODE REVIEW

File: src/Network/Probe.php [hunk 1]
```diff
@@ -305,8 +332,14 @@ public static function uri($uri, $network = "", $uid = 0, $cache = true) {
 			/// @todo temporary fix - we need a real contact update function that updates only changing fields
 			/// The biggest problem is the avatar picture that could have a reduced image size.
 			/// It should only be updated if the existing picture isn't existing anymore.
-			if (($data['network'] != NETWORK_FEED) AND ($mode == PROBE_NORMAL) AND
-				$data["name"] AND $data["nick"] AND $data["url"] AND $data["addr"] AND $data["poll"])
+			if (($data['network'] != NETWORK_FEED)
+				AND ($mode == PROBE_NORMAL)
```
Comment (rabuzarus) [inline src/Network/Probe.php]: Note: `$mode` seems to be uninitialized

File: src/Network/Probe.php [hunk 2]
```diff
@@ -366,9 +399,10 @@ private function detect($uri, $network, $uid) {
 			$nick = ltrim($nick, '@');
 
 			$addr = $nick."@".$host;
+
 		} elseif (strstr($uri, '@')) {
 			// If the URI starts with "mailto:" then jump directly to the mail detection
-			if (strpos($url,'mailto:') !== false) {
+			if (strpos($url, 'mailto:') !== false) {
```
Comment (rabuzarus) [inline src/Network/Probe.php]: Note: `$url` is uninitialized

**Comment by Hypolite** [comment]
Thanks!

File: src/Network/Probe.php (hunk 1 above)
Comment (Hypolite) [inline src/Network/Probe.php]: Look for `probe_url` for the `$mode` parameter.

File: src/Network/Probe.php (hunk 2 above)
Comment (Hypolite) [inline src/Network/Probe.php]: It looks like it should be `$uri`.

**Pull request merged**